File: experiments_dataset_vul_detect_cwe_classification/dataset/vul_dataset/javascript/cwe-89/code_before1.js
// from cwe-snippets, snippets_2/non-compliant/JavaScript/0054.js

var username = document.form.username.value;
var itemName = document.form.itemName.value;
var query = "SELECT * FROM items WHERE owner = " + username + " AND itemname = " + itemName + ";";
db.transaction(function (tx) {
  tx.executeSql(query);
  }
)

File: experiments_dataset_vul_detect_cwe_classification/dataset/vul_dataset/c/cwe-89/code_before1.c
// From cwe-snippets, snippets_2/non-compliant/C/0055.c

ctx.getAuthUserName(?userName); {
CString query = "SELECT * FROM items WHERE owner = '"
     + userName + "' AND itemname = '"
     + request.Lookup("item") + "'";
dbms.ExecuteSQL(query);
